### DISCUSSION TITLE
How would one use zsh-snap to install and configure powerlevel10k prompt

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-snap#31] [Q&A] [answered]
Thanks for your package. I was shocked when I saw how little code there actually was in your source file. I feel like with time I can come to understand it completely. 

I use **powerlevel10K** for my prompt with **antibody** and I'm considering trying out your pluggin manager however I'm not sure how to configure **powerlevel10K**  in my `.zshrc` seeing that it's got a bit more to its configuration than **pure prompt**.

**Comment by marlonrichert** [comment, marked as answer]
Add to your `.zshrc` file:
```zsh
znap source romkatv/powerlevel10k
```
Then restart your shell and follow the instructions in the Powerlevel10k wizard.

**Reply by CatBraaain** [reply]
```shellscript
# .zshrc
[[ -r ~/Repos/znap/znap.zsh ]] || git clone --depth 1 -- https://github.com/marlonrichert/zsh-snap.git ~/Repos/znap
source ~/Repos/znap/znap.zsh

znap source romkatv/powerlevel10k
```

This cause error `(anon):source:27: no such file or directory: /c/Users/username/.local/state/znap/romkatv/internal/p10k.zsh`.

Could you help me please.

*Win10, Git Bash

**Reply by marlonrichert** [reply]
Works fine for me in a clean environment:
```zsh
cd $( mktemp -d )
> .zshrc <<EOF
setopt interactivecomments transientrprompt
PS1='%# '
PS2=
RPS2='%^'
[[ -r ~/Repos/znap/znap.zsh ]] ||
    git clone --depth 1 -- https://github.com/marlonrichert/zsh-snap.git ~/Repos/znap
source ~/Repos/znap/znap.zsh
znap source romkatv/powerlevel10k
EOF
env -i HOME=$PWD PATH=$PATH TERM=$TERM ${TERMINFO:+TERMINFO=$TERMINFO} zsh -d
```
